src/stringProcessing/removePunctuationExceptQuotes.js

```diff
@@ -0,0 +1,18 @@
+// Replace all other punctuation characters at the beginning or at the end of a word.
+var punctuationRegexString = "[\\–\\-\\(\\)_\\[\\]’'.?!:;,¿¡«»‹›\u2014\u00d7\u002b\u0026\<\>]+";
+var punctuationRegexStart = new RegExp( "^" + punctuationRegexString );
+var punctuationRegexEnd = new RegExp( punctuationRegexString + "$" );
+
+/**
+ * Replaces punctuation characters from the given text string.
```

Comment:
`Replaces punctuation characters at the beginning and end of a given text string.`?